src/scaladoc/scala/tools/nsc/doc/model/MemberLookup.scala

```diff
@@ -60,9 +60,9 @@ trait MemberLookup extends base.MemberLookupBase {
         /* Get package object which has associatedFile ne null */
         sym.info.member(newTermName("package"))
       else sym
-    classpathEntryFor(sym1) flatMap { path =>
+    classpathEntryFor(sym1) flatMap { case path =>
```

Comment:
I don't see a reason to add this?

Reply:
Pushed to remove this. I was experimenting with if I could make the if statements clearer but decided against the change and forgot to remove all my changes.